resources/views/projects/milestones/index.blade.php

```diff
@@ -0,0 +1,129 @@
+@extends('layouts.app')
+
+@section('title', 'View milestones')
+
+@section('content')
+
+<div class="padded content">
+	<h1>View milestones</h1>
+
+	<a class="ui button primary right floated" href="">Create</a>
+	<table class="ui celled table">
+		<thead>
+			<tr>
+				<th>#</th>
+				<th>Name</th>
+				<th>Status</th>
+				<th>Estimated date</th>
+				<th>Done date</th>
+				<th>Action</th>
+			</tr>
+		</thead>
+		<tbody>
+			{{-- @foreach($milestones as $milestone)
+			<tr>
+				<td>{{ $loop->iteration }}</td>
+
+				<td>{{ $milestone->name }}</td>
+
+				<td>
+					@switch($milestone->status)
+					@case(Config::get('enum.milestone_status')['done'])
+					<span class="">Done</span>
+					@break
+
+					@case(Config::get('enum.milestone_status')['current'])
+					<span class="">Current</span>
+					@break
+
+					@case(Config::get('enum.milestone_status')['coming-up'])
+					<span class="">Coming up</span>
+					@break
+
+					@default
+					<span class="">Coming up</span>
+					@endswitch
+				</td>
+
+				<td>{{ $milestone->estimated_date }}</td>
+
+				<td>{{ isset($milestone->done_date) ?  $milestone->done_date : '-' }}</td>
+
+				<td>
+					<a class="ui button primary" href="{{ url('/projects/' . $project->id . '/milestones/' . $milestone->id . '/edit') }}">Edit</a>
```

Comment:
Don't manually build the URL. https://laravel.com/docs/5.7/urls#urls-for-controller-actions